proto/api/v1/event_service.proto

```diff
@@ -0,0 +1,60 @@
+syntax = "proto3";
+
+option go_package = "v1";
+
+option java_package = "io.stackrox.proto.api.v1";
+
+import weak "google/api/annotations.proto";
+import "api/v1/common.proto";
+import "api/v1/pagination.proto";
+import "google/protobuf/timestamp.proto";
+
+package v1;
+
+message EventsResponse {
+    repeated Event events      = 1;
+    Pagination     pagination  = 2;
+}
+
+message GetEventsRequest {
+    google.protobuf.Timestamp from      = 1;
+    map<string, string> labels_to_match = 2;
+}
+
+// Events are emitted by Central. They are used to create transparency for users
+// for asynchronous, background tasks.
+// Currently, events only cover logs emitted by Central.
+message Event {
+    // UUID of the event.
+    string id                            = 1;
```

Comment:
I think event should also have a correlation id to query all events pertaining to the same origin. Not sure though if we need an extra field for this or if we should just put a label.

Reply:
Hmm, in my mind those would be under `labels` for now, seeing as this is not generic enough to be a candidate for a standalone field IMO (when we have events emitted from things other than logs, we'll probably don't have correlation IDs and want to be a bit more flexible).